code/obj/artifacts/artifact_items/power_cell.dm

```diff
@@ -20,7 +24,34 @@
 			var/datum/artifact/A = src.artifact
 			src.maxcharge = rand(15,1000)
 			src.maxcharge *= 100
+			src.chargeCap = src.maxcharge
 			A.react_elec[2] = src.maxcharge
+
+			// effects
+			src.effectProbModifier = 1/rand(10,50) 	// probability
+			switch(A.artitype.name)										// noise
+				if ("martian")
+					src.noise = pick('sound/voice/babynoise.ogg', 'sound/voice/animal/bugchitter.ogg', 'sound/voice/blob/blobdeath.ogg', 'sound/voice/farts/frogfart.ogg', 'sound/effects/splort.ogg')
+				if ("ancient")
+					src.noise = pick('sound/effects/electric_shock_short.ogg', 'sound/effects/creaking_metal2.ogg','sound/machines/weapons-reloading.ogg', 'sound/machines/glitch1.ogg','sound/machines/glitch2.ogg', 'sound/machines/glitch3.ogg','sound/machines/glitch4.ogg', 'sound/machines/glitch5.ogg', 'sound/machines/scan.ogg')
+				if ("wizard")
+					src.noise = pick('sound/weapons/airzooka.ogg', 'sound/misc/newsting.ogg', 'sound/misc/chair/glass/scoot5.ogg', 'sound/misc/chair/glass/scoot2.ogg')
+				if ("precursor") // what does precursor stuff even sound like???
+					src.noise = pick('sound/effects/singsuck.ogg', 'sound/effects/screech_tone.ogg')
```

Comment:
It's probably more recommended to use the sound cache. Just change `'` to `"`.

Reply:
Oh darn, I thought I had the right one, I can never remember which it is!